src/bin/ambit/cmd.rs

```diff
@@ -70,6 +72,36 @@ fn get_ambit_paths_from_entry<'a>(
     )
 }
 
+// Recursively search dotfile repository for config path.
+fn get_repo_config_paths() -> Vec<PathBuf> {
+    let mut repo_config_paths = Vec::new();
+    for dir_entry in WalkDir::new(&AMBIT_PATHS.repo.path) {
+        if let Ok(dir_entry) = dir_entry {
+            let path = dir_entry.path();
+            if let Some(file_name) = path.file_name() {
+                if file_name == CONFIG_NAME {
+                    repo_config_paths.push(path.to_path_buf());
+                }
+            }
+        }
+    }
+    repo_config_paths
+}
+
+// Prompt user for confirmation with message.
+fn prompt_confirm(message: &str) -> AmbitResult<bool> {
+    print!("{} [Y/n] ", message);
+    io::stdout().flush()?;
+    let mut answer = String::new();
+    io::stdin().read_line(&mut answer)?;
+    if let Some(answer) = answer.chars().next() {
```

Comment:
This only checks the first character. If my input was `yasjklgfasdjflj` it should not confirm it. (It should only confirm if it is _exactly_ either y or Y.)